SampleApp/javascript/yarn.lock

```diff
@@ -683,10 +683,10 @@
   dependencies:
     "@hapi/hoek" "^9.0.0"
 
-"@iterable/react-native-sdk@^1.1.0":
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/@iterable/react-native-sdk/-/react-native-sdk-1.1.0.tgz#67235d73f2aa3a2c499c7a609d9f748a91c8323d"
-  integrity sha512-My87barDdjDbC5OzPxqaFhOsYukfB8D1wrawG+0MEvgypsWcS5smGdUf8C+xysgvAvsdG2pbcE3AdeTl1OIlLA==
+"@iterable/react-native-sdk@^1.1.1":
+  version "1.1.2"
```

Comment:
Is this supposed to be 1.1.1?

Reply:
Nope, we had release 1.1.2 [here](https://github.com/Iterable/react-native-sdk/releases/tag/1.1.2)